scripts/circle-ci/run-circle-tests.sh

```diff
@@ -82,7 +82,7 @@ if [ "$test_suite_index" -eq "15" ]; then
 elif [ "$test_suite_index" -eq "4" ]; then
     export _JAVA_OPTIONS="-Xms8g -Xmx8g -XX:ActiveProcessorCount=8 ${JAVA_GC_LOGGING_OPTIONS}"
 elif [ "$test_suite_index" -eq "3" ]; then
-    export _JAVA_OPTIONS="-Xms8g -Xmx8g -XX:ActiveProcessorCount=8 ${JAVA_GC_LOGGING_OPTIONS}"
+    export _JAVA_OPTIONS="-Xms16g -Xmx16g -XX:ActiveProcessorCount=8 ${JAVA_GC_LOGGING_OPTIONS}"
```

Comment:
just saw this quickly, does the instance type we are using here have enough memory?

Reply:
It should, as they run on [xlarge resource class](https://github.com/palantir/atlasdb/blob/b79e6fbda020da652b7aecefb1a0a4ce222ae8b9/.circleci/config.yml#L40), which [documentation says should have 32gb of ram](https://circleci.com/product/features/resource-classes/)